pom.xml

```diff
@@ -474,7 +479,30 @@
         <artifactId>maven-release-plugin</artifactId>
         <version>2.5.3</version>
       </plugin>
+
+	  <!-- For all subprojects, read a top-level local.properties file for configuration 
+	       information. This can override the properties needed to, e.g., connect to a DB. -->
```

Comment:
Definitely not. It's only for Maven. It is templated into test files (to set DB connection strings), but should not be templated into anything else. It's one of the places we'll want to make it heroku-happy though, so if you can think of a better way to set DB values, make me happy.